lib/business_time/time_extensions.rb

```diff
@@ -108,6 +108,16 @@ def previous_business_day(time)
         time
       end
 
+      # Returns the next business day not including the given date
+      def next_business_day(time)
+        loop do
+          time = time + 1.day
+          break if time.workday?
+        end
```

Comment:
Loop is not the best option, you could try something safer like: 
```
<last business hour of current day> + 1.business.hour
```
it will return a next business day